app/models/user.rb

```diff
@@ -15,6 +11,10 @@ class User < ApplicationRecord
   validates :name, presence: true, length: { minimum: 3 }
   validates :posts_counter, numericality: { only_integer: true }, comparison: { greater_than_or_equal_to: 0 }
 
+  def admin?
+    role == 'admin'
+  end
+
   def all_posts
     posts.includes([:author]).order(created_at: :desc)
   end
```

Comment:
I was able to register as a user but was unable to log in due to the following error arising when I try to log in. Kindly fix this issue and submit for a re-review.
![image](https://user-images.githubusercontent.com/33268927/187628880-2667bd30-3963-42a8-9777-dcde702bd75a.png)